src/framework/pages/admin/conferencerooms/AbstractRoomBasePage.java

```diff
@@ -0,0 +1,68 @@
+package framework.pages.admin.conferencerooms;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+import framework.selenium.SeleniumDriverManager;
+
+/**
+ * This class represent the Menu for conference rooms, save and cancel button
+ * @author Yesica Acha
+ *
+ */
+public abstract class AbstractRoomBasePage {
+	protected WebDriver driver;	
+	protected WebDriverWait wait;
+
+	@FindBy(linkText="Room Info") 
+	WebElement roomInfoLink;
+
+	@FindBy(linkText="Resource Associations") 
+	WebElement resourceAssociationsLink;
+
+	@FindBy(linkText="Out of Order Planning") 
+	WebElement outOfOrderPlanningLink;
+
+	@FindBy(xpath="//button[@ng-click='save()']") 
+	WebElement saveBtn;
+
+	@FindBy(xpath="//button[@ng-click='cancel()']") 
```

Comment:
out of java convention there should be space between the equal and the variable and its value also